_tutorials/install-add-fdroid-repo.html

```diff
@@ -6,65 +6,70 @@
 ---
 <section>
 
-    <p><span class="text-info">Team NewPipe's F-Droid repository allows for updates to NewPipe to be available faster than on the default F-Droid repository.</span>
+    <p><span class="text-info">Team NewPipe's F-Droid repository allows for updates to NewPipe to be available faster than on the default F-Droid repository. This is the recommended way of installing NewPipe.</span>
```

Comment:
```suggestion
    <p><span class="text-info">Team NewPipe's F-Droid repository allows updates for NewPipe to be available faster than on the default F-Droid repository. This is the recommended way of installing NewPipe.</span>
```

either my english is mixed up or this is correct...

Reply:
Both are correct, but yours is much better.